apis/v1beta1/vspherevm_types.go

```diff
@@ -142,7 +142,7 @@ type VSphereVMStatus struct {
 }
 
 // +kubebuilder:object:root=true
-// +kubebuilder:resource:path=vspherevms,scope=Namespaced
+// +kubebuilder:resource:path=vspherevms,scope=Namespaced,categories=cluster-api
```

Comment:
What's the consquence if missing?

Reply:
Querying all the resources belonging to the cluster-api category would result in the VsphereVM CRs not show up. Mainly would affect any diagnostic bundles, which would used the cluster-api category to pull all the resources belonging to that category.